doc/release-notes-28132.md

```diff
@@ -0,0 +1,5 @@
+Full Replace-By-Fee
```

Comment:
This file is named incorrectly.

Reply:
Release note fragments should have the PR number in them, and this is PR 30493, not 28132. The release version is not relevant for release note fragments.